reconcile/change_owners/change_owners.py

```diff
@@ -370,7 +378,7 @@ def run(
         labels = manage_conditional_label(
             labels=labels,
             condition=self_serviceable and approved,
-            true_label=APPROVED,
+            true_label=get_approved_label_by_priority(change_type_processors),
```

Comment:
`change_type_processors` holds all change-types found in a-i, not just the ones that are active in this MR.

if will find the involved change-types in `changes[*].diff_coverage[*].coverage[*]`